java/src/main/java/ai/rapids/cudf/Cuda.java

```diff
@@ -39,9 +45,9 @@
     protected boolean cleanImpl(boolean logErrorIfNotClean) {
       boolean neededCleanup = false;
       long origAddress = stream;
-      if (stream != 0) {
+      if (stream != defaultStream()) {
         destroyStream(stream);
-        stream = 0;
+        stream = defaultStream();
```

Comment:
Probably simplest to keep using 0 as an indicator the stream has been cleaned, that way we don't have to compute anything and it's consistent how custom streams look cleaned regardless of PTDS setting.

Reply:
Done.